docs/src/pages/start-here.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Start Here
+---
+
+# Start here
+
+## What is this?
+
+[These organizations](about) are working together to define a common set of
+annotation types for use in Java static analysis, beginning with nullness
+analysis.
+
+This includes providing both the JAR file your code can depend on, and *precise*
+specifications of these annotations' semantics.
+
+* **Why standardize the annotations?** This tragic 
+  [stackoverflow answer](https://stackoverflow.com/questions/4963300/which-notnull-java-annotation-should-i-use)
+  says it all.
+* **Why standardize the semantics?** Because a library owner, or a team that
+  uses multiple tools, deserves a single source of truth telling them how to
+  annotate their code, instead of having to decide which tool to make happy.
+* **Why do both together?** Because users deserve annotation types with
+  Javadoc *right there* that answers their questions authoritatively.
+
+Our release will be the *first* tool-neutral, library-neutral artifact for
+these annotations; the first to be developed by *consensus* of the major
+stakeholders. (Note: `javax.annotation` was an aborted attempt that never
+actually released, whose draft work was jammed into Maven anyway.)
+
+Learn more about the JSpecify group and its goals in the [JSpecify
+FAQ](http://github.com/jspecify/jspecify/wiki/jspecify-faq).
+
+## How do I learn about your nullness support?
+
+As you read, if you ever think "that's weird, why did they do that?", have a
+look at the [Nullness Design
+FAQ](https://github.com/jspecify/jspecify/wiki/nullness-design-FAQ); your
```

Comment:
Suggestion: Reference links will make this kind of source easier to read instead of wrapped labels.

Reply:
meant to get to that, will do.